ecosystem/sep-0024.md

```diff
@@ -190,6 +192,73 @@ The deposit flow can only be fulfilled if the Stellar `Account` has established
 5. `Anchor` detects the trust line creation in the `Account`. If the asset is `AUTH_REQUIRED`, `Anchor` approves the new trust line.
 6. `Anchor` proceeds with the deposit flow.
 
+##### Claimable Balances
+
+[Claimable Balances](https://www.stellar.org/blog/protocol-14-improvements) allows payments to accounts that do not have a trust-line for the asset sent.
+
+**Wallets**: It's mandatory for wallets to support Claimable Balances and should poll for their users' received Claimable Balances so that users can "claim" them.
+
+**Anchors**: It's optional for anchors to use submit Claimable Balances as deposit transactions to Stellar accounts that don't yet have a trust-line to the asset they have requested.
+
+**Wallet Claimable Balance Flow**
+
+Wallets will still follow the [typical deposit flow](#Basic-Wallet-Implementation).
+The POST request however will extend to include a `claimable_balance_supported` parameter
+
+**Wallet Claimable Balance Polling**
+
+Claimable Balance Polling allows the wallet to update the claimant that their is a balance available to claim and they may need to establish[trust-line](#stellar-account-doesnt-trust-asset) in order to do so.
```

Comment:
Wallets should probably poll claimable balances as a general feature, but for SEP-24 I think the flow as an isolated experience shouldn't require it. The anchor should provide a claimable balance ID and the wallet should try and claim only that ID. Otherwise the UI/UX could become quite confusing. If for example a claimable balance comes in unrelated to the anchor at the same time for a different amount, that would be very confusing for the user and lead to a support ticket being opened about the other amount. From a security point-of-view we should also avoid arbitrary transactions impacting a well defined user flow as well.

Reply:
>Wallets should probably poll claimable balances as a general feature, but for SEP-24 I think the flow as an isolated experience shouldn't require it. 

I agree that is should be a general feature however it should be a required feature for wallets who intend to support claimable balances as a fallback to communicate with Wallet users that they have balances available.
It's currently implemented [here](https://github.com/stellar/stellar-demo-wallet/pull/49). 
I want to provide wallets as little friction as possible when interacting with anchors who may be still in the process of supporting claimable balances.

> The anchor should provide a claimable balance ID and the wallet should try and claim only that ID. 

We discussed this and we would like to have this feature when anchor adoption of claimable balances are relatively high.
Then we can require anchors send claimable balance IDs in their deposit response that Wallets can anticipate.
So we're attempting to strategize our phases for wallets and anchors not to overwhelm them with too many changes immediately.